src/paginator.js

```diff
@@ -0,0 +1,300 @@
+import assign from 'lodash.assign';
+import omit from 'lodash.omit';
+
+
+export const styles = Object.freeze({
+  pageNumber: 1,
```

Comment:
> This looks like an enum, I generally expect the members to be uppercase

Yup, it is. I agree on uppercase too, though I haven't seen it being used. That's one the things I listed in the listed in the design guide update. I'll mention it there.